fecgov/fec-cms
# 6305 National Party Accounts Types Filters Labels

## Summary

- Resolves #6305 

Updating the labels for the type filters for the National Party Accounts datatables

### Required reviewers

- UX?
- 1 Front-end
- SME

## Impacted areas of the application

The national party accounts datatables' [receipts | disbursements] type filters

## Screenshots
Disbursements
<img width="466" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/dc459f37-8ff9-4851-9565-7d8253608345">

Receipts
<img width="370" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/4b8d352a-9d2e-4d1f-ac13-548d3c542aa6">


## Related PRs

None

## How to test

- pull the branch
- `./manage.py runserver`
- check [national party accounts disbursements](http://127.0.0.1:8000/data/national-party-account-disbursements/)' disbursement type filter (left column, Disbursement details)
- check [national party accounts receipts](http://127.0.0.1:8000/data/national-party-account-receipts/)' receipt type filter (left column, Receipt details)
